sktm/patchwork.py

```diff
@@ -360,20 +365,21 @@ def set_patch_check(self, pid, jurl, result):
             result: Test result (sktm.tresult) to feature in the "check"
                     state.
         """
-        if self.apikey == None:
+        if self.apikey is None:
             logging.debug("No patchwork api key provided, not setting checks")
             return
 
-        payload = { 'patch' : pid,
-                    'state' : None,
-                    'target_url' : jurl,
-                    'context' : 'skt',
-                    'description' : 'skt boot test' }
+        payload = {'patch': pid,
+                   'state': None,
+                   'target_url': jurl,
+                   'context': 'skt',
+                   'description': 'skt boot test'}
         if result == sktm.tresult.SUCCESS:
             payload['state'] = int(pwresult.SUCCESS)
         elif result == sktm.tresult.BASELINE_FAILURE:
             payload['state'] = int(pwresult.WARNING)
-            payload['description'] = 'Baseline failure found while testing this patch'
+            payload['description'] = 'Baseline failure found while testing '
+            'this patch'
```

Comment:
Can we have this indented past the equals sign?

Reply:
pycodestyle will complain about unexpected indentation if we do that. If you have issue with the readability, we can do

```payload['description'] = ('Baseline failure found while testing '
                                          'this patch')
```

but since Python depends on indentation we can't just arbitrarily move the code.